dapp/src/pages/LandingPage/components/CurrentSeasonSection.tsx

```diff
@@ -78,52 +78,55 @@ export default function CurrentSeasonSection() {
           Season 1 stakers will soon be able to earn Acre and Mezo points.
         </TextMd>
 
-        {/* TODO: Uncomment when TVL is higher */}
-        {/* <TextMd fontWeight="semibold" mb={4}>
-          Total value locked
-        </TextMd>
+        {featureFlags.TVL_ENABLED && (
+          <>
+            <TextMd fontWeight="semibold" mb={4}>
+              Total value locked
+            </TextMd>
 
-        <ProgressBar
-          size={{
-            base: "xl",
-            md: "2xl",
-          }}
-          value={seasonProgress}
-          maxW="50rem" // 800px
-        >
-          <CurrencyBalance
-            amount={seasonTotalAssets}
-            currency="bitcoin"
-            variant={{
-              base: "greater-balance-md",
-              md: "greater-balance-xl",
-            }}
-            symbolFontWeight="black"
-            desiredDecimals={2}
-            // TODO: Refactor `CurrencyBalance` to make font styles truely adjustable
-          />
-        </ProgressBar>
+            <ProgressBar
+              size={{
+                base: "xl",
+                md: "2xl",
+              }}
+              value={seasonProgress}
+              maxW="50rem" // 800px
+            >
+              <CurrencyBalance
+                amount={seasonTotalAssets}
+                currency="bitcoin"
+                variant={{
+                  base: "greater-balance-md",
+                  md: "greater-balance-xl",
+                }}
+                symbolFontWeight="black"
```

Comment:
Reverted changes since I will add button with separate PR as requested on Discord.
Here is the new PR with cherry picked commit: https://github.com/thesis/acre/pull/712

Reply:
Ref commit: fd1a952d